scripts/discover/as-ip-resolution

```diff
@@ -29,7 +34,7 @@ if [ ! -f "$RECON_FILE" ] ; then
   mkdir -p "$(dirname $RECON_FILE)"
 
   _ 'Resolving IP Addresses'
-  fast-resolv -c $FR_THREADS -d scope-ips-generated-public*.txt | tee "$RECON_FILE"
+  fast-resolv $FR_RESOLVCONF -c $FR_THREADS -d scope-ips-generated-public*.txt | tee "$RECON_FILE"
```

Comment:
Been meaning to mention this but `fast-resolv` can only take one file at a time right now but this like is trying to pass in multiple via `scope-ips-generated-public*.txt`

Reply:
I'll fix fast-resolv

Reply:
I made fast-resolv read from stdin...